tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/README.md

```diff
@@ -707,12 +707,13 @@ The main difference between this converter and `super_resolution` in data organi
 * `see_in_the_dark` - converts See-in-the-Dark dataset described in the [paper](https://cchen156.github.io/paper/18CVPR_SID.pdf) to `ImageProcessingAnnotation`.
   * `annotation_file` - path to image pairs file in txt format.
 * `conll_ner` - converts CONLL 2003 dataset for Named Entity Recognition to `BERTNamedEntityRecognitionAnnotation`.
-  * `annotation_file` - annotation file in txt forma
+  * `annotation_file` - annotation file in txt format.
   * `vocab_file` - vocab file for word piece tokenization.
   * `lower_case` - converts all tokens to lower case during tokenization (Optional, default `False`).
   * `max_length` - maximal input sequence length (Optional, default 128).
   * `pad_input` - allow padding for input sequence if input less that `max_length` (Optional, default `True`).
   * `include_special_token_lables` - allow extension original dataset labels with special token labels (`[CLS'`, `[SEP]`]) (Optional, default `False`).
+  * `labels_file` - path to file with custom labels in json format (Optional).
```

Comment:
could you please provide info about file format, or link on file example? it is not clear which content expected in json file in labels

Reply:
Done